ui_src/src/components/radioButton/index.js

```diff
@@ -46,9 +46,14 @@ const RadioButton = (props) => {
                 defaultValue={radioValue || options[0]?.value}
             >
                 {options.map((option) => (
-                    <Radio key={option.id} value={option.value}>
-                        <span style={{ fontFamily: fontFamily }}>{option.label}</span>
-                    </Radio>
+                    <>
```

Comment:
I made many changes in this component, so you need to pull from the master first and add your changes.

Reply:
Done